aten/src/ATen/native/native_functions.yaml

```diff
@@ -2132,6 +2132,11 @@
     CPU: kl_div_backward_cpu
     CUDA: kl_div_backward_cuda
 
+- func: kron(Tensor self, Tensor other) -> Tensor
+  variants: function, method
+
```

Comment:
Per our recent dispatch updates, I think this should get an explicit "Math" dispatch key because it's implemented as a composite of existing PyTorch operators.

Reply:
If no dispatch key is added then it is assumed implicitly that the function is of type "Math".

> To use autograd support for training, simply skip adding a dispatch section or write
```
dispatch:
  Math: kernel
```
> You're done. This will allow this op to be correctly registered for both inference and training.

So do we add the dispatch explicitly here?

Reply:
Let's go ahead and add it. The new dispatch system is so new I'm not sure if the standard will be to elide Math dispatches in the future or not, but I'd prefer we give people examples of how it's meant to be used for now.